test/functional/sqlsrv/sqlsrv_ae_azure_key_vault_keywords.phpt

```diff
@@ -114,7 +85,7 @@ for ($i = 0; $i < sizeof($columnEncryption); ++$i) {
                     checkErrors(
                         $errors,
                         array('08001','0'),
-                        array('08001','-1'),    // SSL error occurs in Ubuntu
+                        array('08001','-1'),    // SSL error on some Linuxes
                         array('IMSSP','-110'),
```

Comment:
Do we still have SSL errors after this split?

Reply:
Yes, because I did not split the keywords test. This test runs through 254 connections so splitting is not practical.

Reply:
How often does the SSL error occur? Does it always happen after the first successful connection?

Reply:
I don't remember how often it happens...